fe/lib/components/JobCategorySuggest/JobCategorySuggestChoices.tsx

```diff
@@ -96,6 +101,24 @@ const JobCategorySuggestChoices = forwardRef<HTMLInputElement, Props>(
             );
           })}
         </RadioGroup>
+        {selectedJobCategory === 'Other' && (
```

Comment:
Can `JobCategorySelect` be nested as a child of `<RadioItem label="Other" />` above?

See https://seek-oss.github.io/braid-design-system/components/RadioGroup#toggling-nested-content.